packages/Modal/src/controller/Modal.tsx

```diff
@@ -112,22 +113,22 @@ export const Modal = ({
   overrides,
   ...rest
 }: IModalProps): any => {
+  const target = usePortal({ id: MODAL_PORTAL_ID });
+
   const { container, header, footer } = useComponentLogic({
     componentVariant: variant,
     theme: theme,
     overrides: overrides,
   });
 
-  return (
-    <>
-      <ModalView classes={container} {...rest}>
-        {type !== ModalType.ALERT && <ModalHeader classes={header} title={title} onClose={onClose}></ModalHeader>}
-        {children}
-        {showFooter && <ModalFooter classes={footer} onClose={onClose} actions={actions} type={type}></ModalFooter>}
-      </ModalView>
-      <div className="fixed inset-0 z-40 bg-black opacity-25"></div>
-    </>
-  );
+  return ReactDOM.createPortal(<>
+    <ModalView classes={container} {...rest}>
+      {type !== ModalType.ALERT && <ModalHeader classes={header} title={title} onClose={onClose}></ModalHeader>}
+      {children}
+      {showFooter && <ModalFooter classes={footer} onClose={onClose} actions={actions} type={type}></ModalFooter>}
+    </ModalView>
+    <div className="fixed inset-0 z-40 bg-black opacity-25"></div>
```

Comment:
This looks like its been updated to use tailwind. I don't think we've tested this version in blaze which makes me worry there might be issues. Will discuss in slack